frontend/pulse3d/src/components/interactiveAnalysis/InteractiveAnalysisModal.js

```diff
@@ -169,6 +169,13 @@ export default function InteractiveWaveformModal({
   const [duplicateModalOpen, setDuplicateModalOpen] = useState(false);
   const [creditUsageAlert, setCreditUsageAlert] = useState(false);
   const { usageQuota } = useContext(AuthContext);
+  const [depricationNotice, setDepricationNotice] = useState(false);
+  const [pulse3dVersionDeletionDate, setPulse3dVersionDeletionDate] = useState("");
```

Comment:
`pulse3dVersionDeletionDate` is a little confusing because we aren't actually deleting the versions. This should be `pulse3dVersionEOLDate` so it matches the terminology used in other parts of the code